vector/src/main/java/im/vector/app/features/voicebroadcast/listening/VoiceBroadcastPlayerImpl.kt

```diff
@@ -73,7 +72,7 @@ class VoiceBroadcastPlayerImpl @Inject constructor(
         @MainThread
         set(value) {
             if (field != value) {
-                Timber.w("isLiveListening: $field -> $value")
+                Timber.w("## Voice Broadcast | isLiveListening: $field -> $value")
```

Comment:
This will produce many logs, right? Can we avoid if not necessary?

Reply:
Not so many, it will only log if the user is listening to a voice broadcast.
Then, if the VB is ended, it will log once (or not if the default value does not change). If the VB is live, it will log if the user scrolls in the playback or if the recorder pause/resume the recording, but in a quite acceptable frequency.
Btw, it doesn't need to warn, I changed it to debug

Same for the log about `playingState`, It can be a little more frequent but also only when listening to a VB. The frequency is still acceptable imo, and the log is useful for debugging.
Also changed to debug level